server/routes/punishments/punishmentDaysValidator.ts

```diff
@@ -129,9 +132,17 @@ export default function validatePunishmentDays(
   return null
 }
 
-function formatPrivilegeErrorText(privilegeType: PrivilegeType, href: string, text: string): FormError {
+function formatPrivilegeErrorText(
+  privilegeType: PrivilegeType,
+  otherPrivilege: string,
+  href: string,
+  text: string
+): FormError {
   return {
     href,
-    text: text.replace('[thing]', convertPrivilegeType(privilegeType)),
+    text: text.replace(
+      '[thing]',
+      privilegeType === PrivilegeType.OTHER ? otherPrivilege : convertPrivilegeType(privilegeType)
```

Comment:
Spotted in [PunishmentResult.ts](https://github.com/ministryofjustice/hmpps-manage-adjudications/blob/55f57e7ba5d88b9bb6775c6cbd5eb4bf6e6207f8/server/data/PunishmentResult.ts#L133) we convert to lowercase, here I thought to keep the same case as the user inputted. Happy to change if we want to be consistent 🙂